### PR TITLE
Fix for Suave >= v2.1.0

### PR DESCRIPTION
**Pull request by Badmoonz**

### CODE REVIEW

**Comment by ncthbrt** [comment]
:shipit:

**Comment by ncthbrt** [comment]
@Badmoonz think you need to target dotnet core 2.0 to get the build to work.

File: src/Fable.Websockets.Suave/Fable.Websockets.Suave.fsproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\Meta.props" />
   <PropertyGroup>
     <TargetFramework>netstandard1.6</TargetFramework>
```
Comment (ncthbrt) [inline src/Fable.Websockets.Suave/Fable.Websockets.Suave.fsproj]: This will need to change to v2 I believe

Comment (Badmoonz) [reply src/Fable.Websockets.Suave/Fable.Websockets.Suave.fsproj]: we can keep TargetFramework minimal, until we need some new api

**Comment by Badmoonz** [comment]
hmmm, don't get what's wrong with build
It was just stopped

**Comment by ncthbrt** [comment]
@Badmoonz 
The `.travis.yml` file is installing yarn as it wasn't supported at the time of creating this project. However it is now supported, so removing the following lines from the build file should resolve the issue: 
  ```- sudo apt-key adv --keyserver pgp.mit.edu --recv D101F7899D41F3C3
  - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
  - sudo apt-get update -qq
- sudo apt-get install -y -qq yarn```

**Comment by Badmoonz** [comment]
wow! build passed! thx

**Comment by OlegZee** [comment]
@ncthbrt could you please publish it to nuget so that I can complete migration (make PR) for Fable.Websockets.Elmish?

**Pull request merged**